lib/pure/asynchttpserver.nim

```diff
@@ -132,6 +132,19 @@ proc parseProtocol(protocol: string): tuple[orig: string, major, minor: int] =
 proc sendStatus(client: AsyncSocket, status: string): Future[void] =
   client.send("HTTP/1.1 " & status & "\c\L\c\L")
 
+proc parseUppercaseMethod(name: string): HttpMethod =
+  result = case name
+  of "GET": HttpGet
```

Comment:
Please align the ``of`` under the ``case`` keyword.

Reply:
To be clear:

```nim
result =
  case name
  of "GET": ...
```